dash/development/_py_components_generation.py

```diff
@@ -1,7 +1,7 @@
 from collections import OrderedDict
 import copy
 import os
-from textwrap import fill
+from textwrap import fill, dedent
```

Comment:
did `dedent` end up being used?

Reply:
We tried to use it. But it didn't work as intended 😅 
Removed at https://github.com/plotly/dash/pull/2218/commits/1c92af470413f5f323639bceb4e425668ec98afa